src/__tests__/BoardSizeForm.test.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import BoardSizeForm from '../components/BoardSizeForm';
+
+describe('BoardSizeForm component', () => {
+  it('Renders a form', () => {
+    const wrapper = shallow(
+      <BoardSizeForm
+        handleChange={jest.fn()}
+        handleSubmit={jest.fn()}
+        hasError={false}
+      />
+    );
+    expect(wrapper.exists('#size-form')).toBeTruthy();
+  });
+
+  it('Renders an error if given one', () => {
+    const wrapper = mount(
+      <BoardSizeForm
+        handleChange={jest.fn()}
+        handleSubmit={jest.fn()}
+        hasError={true}
+      />
+    );
+    expect(
+      wrapper.contains(
+        <p>Error: Invalid input. Please input a number from 3 - 10</p>
+      )
+    ).toBeTruthy();
+  });
+
+  it('Calls its handleChange method', () => {
+    const mockFn = jest.fn();
+    const wrapper = shallow(
+      <BoardSizeForm
+        handleChange={mockFn}
+        handleSubmit={jest.fn()}
+        hasError={false}
+      />
+    );
+    expect(mockFn.mock.calls).toHaveLength(0);
+    wrapper
+      .find('input')
```

Comment:
Could you add a class or specific ID to the input you are testing so it doesn't rely on plucking randomly? I find that to be fragile